cubedash/_filters.py

```diff
@@ -15,7 +15,7 @@
 from dateutil import tz
 from flask import Blueprint
 from markupsafe import Markup, escape
-from orjson import orjson
+from orjson.orjson import dumps
```

Comment:
```suggestion
import orjson
```
Follow the examples in the [orjson docs](https://pypi.org/project/orjson/), and also the common usage of the `json` builtin module.

Reply:
Fixed.